src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerAdminPostAproveDeletionIntTest.java

```diff
@@ -111,6 +116,14 @@ void testTranscriptionDocumentHideSuccess() throws Exception {
         TranscriptionDocumentHideResponse transcriptionResponse
             = objectMapper.readValue(mvcResult.getResponse().getContentAsByteArray(), TranscriptionDocumentHideResponse.class);
 
+        List<AuditEntity> caseExpiredAuditEntries = dartsDatabase.getAuditRepository()
+            .findAll((Specification<AuditEntity>) (root, query, criteriaBuilder) -> criteriaBuilder.and(
+                criteriaBuilder.equal(root.get(AuditEntity_.additionalData), String.valueOf(adminActionEntity.getId())),
+                criteriaBuilder.equal(root.get(AuditEntity_.auditActivity).get("id"), AuditActivity.MANUAL_DELETION.getId())
+            ));
+
+        assertFalse(caseExpiredAuditEntries.isEmpty());
```

Comment:
Should validate the content of the audit entry and ensuring you are only getting the expected number of them added

Reply:
Assumed this was good enough seeing as all we control was the activity and the additional data. The query looks specifically for those values so would return an empty list otherwise.

Reply:
Is this approach not consistent with the approach taken https://github.com/hmcts/darts-api/pull/2053

Reply:
I think we should validate the content of the audits just to make sure we have returned the correct audit entries

Reply:
Added additional assertions

Reply:
I do not see the assertions?

Reply:
I do not see these assertions?

Reply:
They are there if you look at the files changed view

Reply:
I can see these now unsure why they were not coming up before. 
My only suggestion would be to add a size() check to ensure you only have one item :)

Reply:
Done